test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -1327,6 +1327,48 @@ def create_model():
             model_type=self.model_type,
         )
 
+    @pytorch_test_common.xfail_if_model_type_is_not_exportedprogram(
+        "DynamoExport does not support input mutation"
```

Comment:
Why in the original test, it doesn't pop this during Dynamo?

Reply:
What do you mean? Can you elaborate?

Reply:
Yes, so the reason huggingface gpt2 is in `test_fx_to_onnx.py` is because we observed errors when we ran the exported model with ORT, but didn't see any issue with dynamo export.

https://github.com/pytorch/pytorch/blob/686a3e0bf08af50a3b83532058fc5a6667365e51/test/onnx/test_fx_to_onnx.py#L531-L556

However, this xfail reason says there is something wrong during dynamo export. I am wondering what's the difference could make the same models/weights having different error/issue.